automationTest/src/test/java/com/owncloud/android/test/ui/models/FileDetailsView.java

```diff
@@ -32,8 +32,8 @@
 	final AndroidDriver driver;
 	
 	@CacheLookup
-	@AndroidFindBy(name = "Keep file up to date")
-	private AndroidElement keepFileUpToDateCheckbox;
+	@AndroidFindBy(name = "Favorite")
```

Comment:
Is the target of this PR the 1.7.2 version ?  The change to 'Favorite' was later.